msteams-platform/bots/how-to/conversations/create-a-bot-commands-menu.md

```diff
@@ -1,12 +1,167 @@
 ---
 title: Create a command menu for your bot
-author: clearab
+author: WashingtonKayaker
 description: How to create a command menu for your Microsoft Teams bot
-ms.topic: overview
+ms.topic: overview, command menu
 ms.author: anclear
 ---
-# Create a command menu for your bot
+# Bot command menus
 
+Creating a command menu enables you to give your users a list of commands that your bot is designed to execute, along with simple help text associated with each command that you can use to explain the command. The user can select the command from the list and it is placed in the message, then all they need to do is click __Send__.
+
+![Bot command menu](Media\bot-menu-sample.png)
+
+
+## Create a command menu for your bot
+
+Command menus are contained in your App Manifest, which can be created in <!--NEED LINK--> [App Studio](https://www.sadanduseless.com/funniest-animals-of-2019/), or edited directly in your source code in the `manifest.json` file in the `TeamsAppManifest` folder. In this section we will demonstrate how to create the command menu in App Studio, followed by the resulting `manifest.json` file so you can choose the approach that best suites your needs.
+
+### Creating a command menu for your bot using App Studio
+
+The instructions here assume that you will be editing an existing app manifest, however the steps of adding a command menu are the same, whether you are creating a new manifest or editing an existing manifest.
```

Comment:
Use "existing app manifest. The steps" or "existing app manifest. However, the steps"